lib/puppet/feature/iis_compatible_version_installed.rb

```diff
@@ -0,0 +1,43 @@
+require 'puppet/util/feature'
+
+# Consider joining MajorVersion + MinorVersion instead of using VersionString
+
+def read_registry_for_iis_compatible_version
```

Comment:
Since this returns a boolean, should rename method to `is_iis_compatible?` or similar.

Reply:
Done